### DISCUSSION TITLE
backend error

### DISCUSSION THREAD

**Discussion by Ivanmecina** [akeeba/panopticon#802] [Q&A (Community Support)] [answered]
This is what I am seeing in my backend of Panopticon for 2 Joomla sites on the site information page. It is not showing any extentions as it was before the update to 1.2
Error: Value of type null is not callable /home/immiivan/pan/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php:423 Stack trace: #0 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1301): include() #1 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile() #2 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #3 /home/immiivan/pan/tmp/compiled_templates/6/d/6d91b45a687f386f9cdc45426b44afbb1d77bdea.php(112): Awf\Mvc\View->loadAnyTemplate() #4 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1301): include('...') #5 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile() #6 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #7 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate() #8 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate() #9 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #10 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(113): Awf\Mvc\Controller->display() #11 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->read() #12 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute() #13 /home/immiivan/pan/src/Controller/Sites.php(64): Awf\Mvc\DataController->execute() #14 /home/immiivan/pan/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Sites->execute() #15 /home/immiivan/pan/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #16 /home/immiivan/pan/vendor/akeeba/awf/src/Application/Application.php(247): Akeeba\Panopticon\Dispatcher->dispatch() #17 /home/immiivan/pan/src/Application.php(430): Awf\Application\Application->dispatch() #18 /home/immiivan/pan/index.php(35): Akeeba\Panopticon\Application->dispatch() #19 /home/immiivan/pan/index.php(38): {closure}() #20 {main} Next RuntimeException: Error calling repeatable "extUpdateExtensionIcon" in /home/immiivan/pan/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php:423 Stack trace: #0 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1301): include() #1 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile() #2 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #3 /home/immiivan/pan/tmp/compiled_templates/6/d/6d91b45a687f386f9cdc45426b44afbb1d77bdea.php(112): Awf\Mvc\View->loadAnyTemplate() #4 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1301): include('...') #5 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile() #6 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #7 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate() #8 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate() #9 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #10 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(113): Awf\Mvc\Controller->display() #11 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->read() #12 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute() #13 /home/immiivan/pan/src/Controller/Sites.php(64): Awf\Mvc\DataController->execute() #14 /home/immiivan/pan/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Sites->execute() #15 /home/immiivan/pan/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #16 /home/immiivan/pan/vendor/akeeba/awf/src/Application/Application.php(247): Akeeba\Panopticon\Dispatcher->dispatch() #17 /home/immiivan/pan/src/Application.php(430): Awf\Application\Application->dispatch() #18 /home/immiivan/pan/index.php(35): Akeeba\Panopticon\Application->dispatch() #19 /home/immiivan/pan/index.php(38): {closure}() #20 {main} Next ErrorException: Error calling repeatable "extUpdateExtensionIcon" [View template: ROOT/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php line 423, compiled from ROOT/ViewTemplates/Sites/item_extensions.blade.php] in /home/immiivan/pan/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php:423 Stack trace: #0 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1235): Awf\Mvc\View->handleViewException() #1 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #2 /home/immiivan/pan/tmp/compiled_templates/6/d/6d91b45a687f386f9cdc45426b44afbb1d77bdea.php(112): Awf\Mvc\View->loadAnyTemplate() #3 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1301): include('...') #4 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile() #5 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #6 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate() #7 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate() #8 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #9 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(113): Awf\Mvc\Controller->display() #10 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->read() #11 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute() #12 /home/immiivan/pan/src/Controller/Sites.php(64): Awf\Mvc\DataController->execute() #13 /home/immiivan/pan/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Sites->execute() #14 /home/immiivan/pan/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #15 /home/immiivan/pan/vendor/akeeba/awf/src/Application/Application.php(247): Akeeba\Panopticon\Dispatcher->dispatch() #16 /home/immiivan/pan/src/Application.php(430): Awf\Application\Application->dispatch() #17 /home/immiivan/pan/index.php(35): Akeeba\Panopticon\Application->dispatch() #18 /home/immiivan/pan/index.php(38): {closure}() #19 {main} Next ErrorException: Error calling repeatable "extUpdateExtensionIcon" [View template: ROOT/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php line 423, compiled from ROOT/ViewTemplates/Sites/item_extensions.blade.php] in /home/immiivan/pan/tmp/compiled_templates/5/7/57f3e859f9abf6e9cab132875492f253d0863d24.php:423 Stack trace: #0 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(1235): Awf\Mvc\View->handleViewException() #1 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate() #2 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate() #3 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate() #4 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #5 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(113): Awf\Mvc\Controller->display() #6 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->read() #7 /home/immiivan/pan/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute() #8 /home/immiivan/pan/src/Controller/Sites.php(64): Awf\Mvc\DataController->execute() #9 /home/immiivan/pan/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Sites->execute() #10 /home/immiivan/pan/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #11 /home/immiivan/pan/vendor/akeeba/awf/src/Application/Application.php(247): Akeeba\Panopticon\Dispatcher->dispatch() #12 /home/immiivan/pan/src/Application.php(430): Awf\Application\Application->dispatch() #13 /home/immiivan/pan/index.php(35): Akeeba\Panopticon\Application->dispatch() #14 /home/immiivan/pan/index.php(38): {closure}() #15 {main}

**Comment by nikosdion** [comment, marked as answer]
When pasting error messages or code please put it inside a code fence (three backticks at the top and three backticks at the bottom). If I need to spend 10-15 minutes to add line breaks to read the debug log, I'm sorry, I will not do it.

All I can suggest is delete the contents of your `tmp` folder and reload the page.

**Reply by Ivanmecina** [reply]
Deleting tmp worked thanks
